diff -up pybind11-2.0.1/tests/CMakeLists.txt.tests pybind11-2.0.1/tests/CMakeLists.txt
--- pybind11-2.0.1/tests/CMakeLists.txt.tests	2017-01-04 06:17:18.000000000 -0800
+++ pybind11-2.0.1/tests/CMakeLists.txt	2017-02-24 06:30:19.356044843 -0800
@@ -27,7 +27,7 @@ set(PYBIND11_TEST_FILES
   test_modules.cpp
   test_multiple_inheritance.cpp
   test_numpy_array.cpp
-  test_numpy_dtypes.cpp
+#  test_numpy_dtypes.cpp
   test_numpy_vectorize.cpp
   test_opaque_types.cpp
   test_operator_overloading.cpp
