diff -up pybind11-2.2.2/pybind11/__init__.py.orig pybind11-2.2.2/pybind11/__init__.py
--- pybind11-2.2.2/pybind11/__init__.py.orig	2018-02-07 10:59:22.000000000 +0100
+++ pybind11-2.2.2/pybind11/__init__.py	2018-04-16 11:52:24.631585674 +0200
@@ -2,10 +2,4 @@ from ._version import version_info, __ve
 
 
 def get_include(*args, **kwargs):
-    import os
-    try:
-        from pip import locations
-        return os.path.dirname(
-            locations.distutils_scheme('pybind11', *args, **kwargs)['headers'])
-    except ImportError:
-        return 'include'
+    return '/usr/include/pybind11'
