diff -up pybind11-2.0.1/tests/test_kwargs_and_defaults.py.699 pybind11-2.0.1/tests/test_kwargs_and_defaults.py
diff -up pybind11-2.0.1/tests/test_numpy_array.py.699 pybind11-2.0.1/tests/test_numpy_array.py
--- pybind11-2.0.1/tests/test_numpy_array.py.699	2017-02-26 13:32:20.014729539 -0800
+++ pybind11-2.0.1/tests/test_numpy_array.py	2017-02-26 13:34:13.621804603 -0800
@@ -6,7 +6,7 @@ with pytest.suppress(ImportError):
 
 @pytest.fixture(scope='function')
 def arr():
-    return np.array([[1, 2, 3], [4, 5, 6]], '<u2')
+    return np.array([[1, 2, 3], [4, 5, 6]], '=u2')
 
 
 @pytest.requires_numpy
@@ -82,10 +82,10 @@ def test_dim_check_fail(arr):
                           ([1, 2], [6])])
 def test_data(arr, args, ret):
     from pybind11_tests.array import data, data_t
+    from sys import byteorder
     assert all(data_t(arr, *args) == ret)
-    assert all(data(arr, *args)[::2] == ret)
-    assert all(data(arr, *args)[1::2] == 0)
-
+    assert all(data(arr, *args)[(0 if byteorder == 'little' else 1)::2] == ret)
+    assert all(data(arr, *args)[(1 if byteorder == 'little' else 0)::2] == 0)
 
 @pytest.requires_numpy
 def test_mutate_readonly(arr):
diff -up pybind11-2.0.1/tests/test_numpy_dtypes.py.699 pybind11-2.0.1/tests/test_numpy_dtypes.py
