diff -up pybind11-2.10.0/pybind11/commands.py.hpath pybind11-2.10.0/pybind11/commands.py
--- pybind11-2.10.0/pybind11/commands.py.hpath	2022-08-02 21:10:36.582025308 +0000
+++ pybind11-2.10.0/pybind11/commands.py	2022-08-02 21:11:27.826124432 +0000
@@ -8,18 +8,11 @@ def get_include(user: bool = False) -> s
     Return the path to the pybind11 include directory. The historical "user"
     argument is unused, and may be removed.
     """
-    installed_path = os.path.join(DIR, "include")
-    source_path = os.path.join(os.path.dirname(DIR), "include")
-    return installed_path if os.path.exists(installed_path) else source_path
+    return '/usr/include/pybind11'
 
 
 def get_cmake_dir() -> str:
     """
     Return the path to the pybind11 CMake module directory.
     """
-    cmake_installed_path = os.path.join(DIR, "share", "cmake", "pybind11")
-    if os.path.exists(cmake_installed_path):
-        return cmake_installed_path
-
-    msg = "pybind11 not installed, installation required to access the CMake files"
-    raise ImportError(msg)
+    return '/usr/share/cmake/pybind11'
